src/plugins/assetUploader/irys.ts

```diff
@@ -195,4 +196,24 @@ export class IrysAssetUploader implements IAssetUploader {
       throw new Error(`Error uploading folder to irys ${e}`);
     }
   }
+
+  /**
+   * Get cost estimate to upload
+   *
+   * @param args.account The account to sign the transaction
+   * @param args.numBytes  The number of bytes to check the price for
+   *
+   * @return Cost to upload numBytes, unit is the token specified
+   * when instantiating the Irys object Return value is in atomic units
+   */
+  async getPrice(args: { account: Account; numBytes: number }): Promise<BigNumber> {
+    const { account, numBytes } = args;
+    const irys = await this.getIrys({ account });
+
+    try {
+      return await irys.getPrice(numBytes);
```

Comment:
I would just return a `number` here and remove the use of `BigNumber`

```suggestion
      return await irys.getPrice(numBytes).toNumber;
```